docs/quick-start.md

```diff
@@ -1,19 +1,20 @@
 # Quick Start
+In this example we will send 1 _AE_ coin from one account to another
 
 ## 1. Specify imports
 For the following snippets in the guide you need to specify multiple imports. Most imports like `Universal`, `MemoryAccount` & `Node` are [Stamps](https://stampit.js.org/essentials/what-is-a-stamp) that compose certain functionalities. Others like `AmountFormatter` & `Crypto` are util modules with typical function exports.
 
 ```js
-import {
+const {
```

Comment:
I assume the developers will create `somePlainJsFile.js` and start coding

Personally, I would also use modern import, but this older one works out of the box.

Reply:
Do we want to explain it in plain JS context or in Vue context?

Reply:
We can't specific to one javascript platform. This is good, plain javascript!

Reply:
So this - older ES5 import will work out of the box with plain js

Reply:
Yes, that was my point.
In the modern type of import, there are some `if`s and ` requires a bit more experience` as you mentioned.
For that reason, I would suggest the older style of import as this is straightforward.
But feel free to suggest (describe) the way how to guide our users to use the modern way. 
I am open to it, but I preferred fewer steps in the guide.